test/scales/linearScaleTests.ts

```diff
@@ -349,6 +349,12 @@ describe("Scales", () => {
     });
 
     describe("Plot interaction", () => {
+      function createMockDrawer(dataset: Plottable.Dataset) {
```

Comment:
Had to move it here, because `createMockDrawer` used to be declared in the global namespace. Previously these tests referenced it from there, so when I deleted it from the global namespace I had to move it inside this `describe()`.